lib/code_corps_web/views/task_list_view.ex

```diff
@@ -1,11 +1,21 @@
 defmodule CodeCorpsWeb.TaskListView do
   @moduledoc false
   use CodeCorpsWeb, :view
-  use JaSerializer.PhoenixView
+  use JSONAPI.View, type: "task-list"
 
-  attributes [:done, :inbox, :name, :order, :pull_requests, :inserted_at, :updated_at]
+  alias CodeCorpsWeb.{ProjectView, TaskView}
 
-  has_one :project, type: "project", field: :project_id
+  def render("index.json-api", %{data: task_list, conn: conn}) do
+    __MODULE__.index(task_list, conn, nil)
+  end
 
-  has_many :tasks, serializer: CodeCorpsWeb.TaskView, identifiers: :always
+  def render("show.json-api", %{data: task_list, conn: conn, params: params}) do
+    __MODULE__.show(task_list, conn, params)
+  end
```

Comment:
Would rather just call the show/index action directly from the controller, but was getting failing controller tests around `expected connection to have a response but no response was set/sent.  Please verify that you assign to "conn" after a request`.  Will have to investigate more.